File: create-task1-dirs.js
const fs = require('fs');
const path = require('path');
const { execSync } = require('child_process');

// Create directories
const dirs = ['src/services', 'tests/services'];
dirs.forEach(dir => {
  const fullPath = path.join(process.cwd(), dir);
  fs.mkdirSync(fullPath, { recursive: true });
  console.log('✓ Created:', fullPath);
});

console.log('\nDirectories created successfully!');

File: create-task1-dirs.py
import os

# Create directories
dirs = ['src/services', 'tests/services']
for dir_path in dirs:
    os.makedirs(dir_path, exist_ok=True)
    print(f'✓ Created: {os.path.abspath(dir_path)}')

print('\nDirectories created successfully!')
